yoyodyne/models/lstm.py

```diff
@@ -342,8 +342,8 @@ def add_argparse_args(parser: argparse.ArgumentParser) -> None:
             "--bidirectional",
             action="store_true",
             default=defaults.BIDIRECTIONAL,
-            help="Uses a bidirectional encoder "
-            "(LSTM-backed architectures only. Default: %(default)s.",
+            help="Uses a bidirectional encoder (LSTM-backed architectures "
+            "only. Default: enabled.",
```

Comment:
So if one was to change `defaults.BIDIRECTIONAL,` then one would need to remember to update this text. Feels slightly dangerous. Then again, parsing defaults feels maybe a bit excessive?

Reply:
That's true but I find the additional information (that this is the default value of a boolean flag) useful enough to justify the small potential for debt. 

There is a tradition I learned about at Google of writing programmatic comments which express the notion "if you change this you have to change that"; it's then enforced via some kind of software nudge (looks like your PR changes this, are you sure you don't want to change that?). But I don't know if anything like that exists outside of that context.